### PR TITLE
Update connectors section and minor fixes 

### PR DESCRIPTION
**Pull request by altafan**
This updates the description of the connectors key concept.

It also updates other definitions, please tell me if you want any change reverted.

This also fixes code examples and CLI commands usage.

Please @tiero review.

### CODE REVIEW

File: docs/learn/concepts.md [hunk 1]
```diff
@@ -39,46 +44,53 @@ All time periods used on timelocks (**5s**, **24h**, **4w**, **1y**) are arbitra
...
+
 ### Connectors
 
-To ensure atomicity—preventing users from losing their VTXOs without confirmation of the new ones—the forfeit transaction is contingent on the new shared UTXO transaction. This is achieved by using connector outputs, which are part of the shared UTXO transaction and serve as inputs for the forfeit transactions.
+Connectors are used to ensure atomicity when forfeiting VTXOs in exchange for others in a new VTXO tree, ie. spending a VTXO by joining a round.
+
+As introduced before, a round transaction typically has two outputs: a _Shared Output_ that commits to a VTXO tree, and a _Connector Output_ that commits to a chain of connectors.
+
+A connector is a dust value ouptut that _connects_ a forfeit tx to the round tx in which a VTXO is spent. A connector is simply used as input of a forfeit tx and is signed by the ASP only. It's purpose is to force the forfeit tx to be broadcastable only if the round one is broadcasted as well. This is a guarantee for the user that there's no way the ASP can use his signatures to double-spend a VTXO.
```
[the remark this answers is not in the document]
Comment (brg444) [reply docs/learn/concepts.md:88]: >A connector is a dust value ouptut that _connects_ a forfeit tx to the round tx in which a VTXO is spent

Shouldn't this say rather a "VTXO is *created*"?

[the remark this answers is not in the document]
Comment (tiero) [reply docs/learn/concepts.md:88]: It links both the round where the new VTXO is created and the round where it is spent/destroyed?

[the remark this answers is not in the document]
Comment (brg444) [reply docs/learn/concepts.md:88]: It's definitely create IMO. 

The forfeit transactions is the "spend" and it's connected to the round transaction that creates a new VTXO.

Especially in the context of receivers joining rounds.

File: docs/learn/concepts.md [hunk 2]
```diff
@@ -5,13 +5,18 @@ toc_max_heading_level: 5
 ---
 
 - [🕴️ Actors](#️-actors)
+  - [ASP](#asp)
```
[the remark this answers is not in the document]
Comment (altafan) [reply docs/learn/concepts.md]: Replaced all occurrences of ASP with Server

File: docs/learn/concepts.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (altafan) [reply docs/learn/concepts.md]: Done

**Pull request merged**